corehq/motech/README.md

```diff
@@ -107,25 +106,28 @@ with OpenMRS.
 
 ### Importing data from OpenMRS to CommCare
 
-CommCare can import data from OpenMRS using OpenMRS's Reporting API and
-OpenMRS's Atom feed API. The Reporting API is used for periodic imports,
-for example monthly. The Atom feed API is for importing data as it is
-added or changed in OpenMRS.
+MOTECH can import data into CommCare from OpenMRS using OpenMRS's
+Reporting API and OpenMRS's Atom feed API. The Reporting API is used for
+periodic imports, for example monthly. The Atom feed API is for
+importing data as it is added or changed in OpenMRS.
 
 ### Sending data from CommCare to OpenMRS
 
-CommCare sends data to OpenMRS using its Web Services API.
+MOTECH sends CommCare data to OpenMRS using OpenMRS's Web Services API.
 
 All data sent to OpenMRS relates to what OpenMRS refers to as
-"patients", "visits", "encounters" and "observations". In CommCare these
-correspond to properties of one or a handful of case types, and values
-of form questions.
+"patients", "visits", "encounters" and "observations". Values from
+CommCare can be retrieved from case properties, form questions,
+locations, *inter alia*.
```

Comment:
> inter alia

nescio hoc verbo

Reply:
lol, sorry, I'll change it to "among others" or "etc."